CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Changelog
 Version numbers follow [Semantic Versioning](https://semver.org/).
 
+## v3.1.0
```

Comment:
If anything, if we were to follow semantic versioning properly, we'd update to v4.0.0, as far as I understand things, no?

Reply:
Yeah, that's how Semantic Versioning works... my bad there. Updated the version number to 4.0.0.

Though this does mean that it would be a major version bump every time the metrics crate updates. I wonder if there's any better way to handle those updates?

Reply:
Here's hoping the metrics crate stabilizes soon (releases version 1.0.0), after that version bumps that stay within 1.x.x won't be a breaking update